### ISSUE TITLE
Add the ability to help translate

### ISSUE THREAD

**Issue by MXC48** [aerotoad/Thud#30]
Hello I am Ok to translate the application in my language (French) on the condition that it is added on crowdin, weblate or transifex.
I think it would be the best way to contribute to a promising project 😁

**Comment by shuvashish76** [comment]
Agreed, IMO Weblate is better as it's [opensource](https://github.com/WeblateOrg/weblate).

Features: https://weblate.org/en/features/

![Screenshot_2021-12-05-21-15-09](https://user-images.githubusercontent.com/48226312/144753433-4fc61e84-f12d-42d0-885e-1b9abd399061.jpeg)

https://weblate.org/en/hosting/

**Comment by aerotoad** [comment]
I have to do some research before jumping into this, since right now all of the text is hardcoded into the html.
Before being able to integrate it with a service like those and begin the translation process, all of the strings should be extracted and a localization engine such as transloco installed.

Also, I'm not sure if transloco or the built-in angular localization module are compatible with those platforms, I guess they are but I've never worked with them before, maybe you know more than be and can shed some light on the issue 😅 .

**Comment by shuvashish76** [comment]
Ping @comradekingu for help...

**Comment by comradekingu** [comment]
Depends what format it produces https://docs.weblate.org/en/latest/formats.html#translation-types-capabilities

I can help if you add "kingu" after setting up shop at https://weblate.bubu1.eu/ or https://hosted.weblate.org/hosting/ :)

**Comment by comradekingu** [comment]
Hello person with the same name as "product manager" from Crowdin.
Crowdin is certainly _not_ free for open source projects.
![bilde](https://user-images.githubusercontent.com/13802408/172693901-fabbac2d-c4d5-423b-bed6-2b67c1d63243.png)
Since a project specifically non-profit in that it is non-commercial can't be open source, this means it actually requires projects to have no source of income, or profit motive.
Moreover, Crowdin itself is not free as in open source/libre/Free Software. It is non-free.
The terms of service further invalidate the claim that Crowdin itself for any possible way one could use it is in any way free of cost, since it is an exchange of data users have to be 18 to accept, leaking and being sent to a myriad of companies, for all sorts of commercial reasons.

For example:
https://support.crowdin.com/privacy-policy/
>behaviourally targeted advertising cookies - These cookies are used to deliver adverts more relevant to you and your interests. They are also used to limit the number of times you see an advertisement as well as help measure the effectiveness of the advertising campaigns. They are usually placed by advertising networks with the website operator’s permission. They remember that you have visited a website and this information is shared with other organisations such as advertisers. Quite often targeting or advertising cookies will be linked to site functionality provided by the other organisation.

In my jurisdiction your statements qualify for false advertisement.

**Comment by marciozomb13** [comment]
Using Crowdin and a free license based on your project for my in-house translation.
![image](https://github.com/user-attachments/assets/008ad49c-c00f-47f4-a6e6-009ecdd34d69)
